pkgs/top-level/all-packages.nix

```diff
@@ -4517,7 +4519,11 @@ in
 
   nylon = callPackage ../tools/networking/nylon { };
 
-  nxproxy = callPackage ../tools/admin/nxproxy { };
+  nxproxy = throw ''
```

Comment:
are there any major differences between this `nxproxy` and the `nxproxy` from `nx-libs`? Unless I'd add an alias to `pkgs/top-level/aliases.nix` and probably throw a deprecation warning.

Reply:
Good idea! How does one throw in a deprecation warning?

Reply:
I'd recommend to do something like this:

``` nix
{
  # pkgs/top-level/aliases.nix
  nxproxy = lib.warn "Deprecated. <explanation>" nx-libs; # yyyy-mm-dd
}
```